src/phone-input/country-select.js

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+
+import Flag from './flag.js';
+import {SIZE, countries} from './constants.js';
+import CountrySelectDropdown from './country-select-dropdown.js';
+import {Select as DefaultSelect} from '../select/index.js';
+import {getOverrides, mergeOverrides} from '../helpers/overrides.js';
+
+import type {CountrySelectPropsT} from './types.js';
+
+export default function CountrySelect(props: CountrySelectPropsT) {
+  const {
+    countryValue,
+    inputRef,
+    onCountryChange = event => {},
+    size = SIZE.default,
+    dropdownWidth = '400px',
+    dropdownHeight = '400px',
+    mapIsoToLabel,
+    overrides = {
+      CountrySelect: {},
+      CountrySelectDropdown: {},
+      CountrySelectDropdownListItem: {},
+      CountrySelectDropdownFlagColumn: {},
+      CountrySelectDropdownNameColumn: {},
+      CountrySelectDropdownDialcodeColumn: {},
+    },
+  } = props;
+  const baseOverrides = {
+    ControlContainer: {
+      style: props => {
+        if (!props.$isFocused && !props.$isPseudoFocused) {
+          return {
+            backgroundColor: 'transparent',
+            borderColor: 'transparent',
+          };
+        }
+      },
+    },
+    ValueContainer: {
+      style: {
+        width: {
+          [SIZE.compact]: '34px',
+          [SIZE.default]: '42px',
+          [SIZE.large]: '50px',
```

Comment:
similar comment to the `flags.js` file. why define the values here rather than in theme, and may be more readable if we don't immediately access the property on map